src/Compilers/Core/CodeAnalysisTest/AnalyzerFileReferenceTests.cs

```diff
@@ -247,6 +247,7 @@ public class TestAnalyzer : DiagnosticAnalyzer
                 // Test analyzer load failure.
                 var remoteTest = (RemoteAnalyzerFileReferenceTest)loadDomain.CreateInstanceAndUnwrap(typeof(RemoteAnalyzerFileReferenceTest).Assembly.FullName, typeof(RemoteAnalyzerFileReferenceTest).FullName);
                 var exception = remoteTest.LoadAnalyzer(analyzerFile.Path);
+                Console.WriteLine(exception);
```

Comment:
>WriteLine [](start = 24, length = 9)

Is this intentional?

Reply:
Ugg. I thought I deleted all of the diagnostic code. Let me fix.